content/oci/storage.go

```diff
@@ -107,6 +110,18 @@ func (s *Storage) Push(_ context.Context, expected ocispec.Descriptor, content i
 	return nil
 }
 
+// Return the ingest file matching name
```

Comment:
Nit: public functions doc comments should follow convention of starting with function name.